CMakeLists.txt

```diff
@@ -20,30 +20,35 @@
 
 cmake_minimum_required(VERSION 3.5)
 
+include(GNUInstallDirs)
+
 add_subdirectory(media_driver)
 
 if (DEFINED BUILD_ALONG_WITH_CMRTLIB)
     add_subdirectory(cmrtlib)
 endif()
 
 if (TARGET iHD_drv_video AND TARGET igfxcmrt)
-    install (FILES ${CMAKE_CURRENT_BINARY_DIR}/media_driver/iHD_drv_video.so DESTINATION /usr/lib/x86_64-linux-gnu/dri/)
-    install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/media_driver/cmake/linux/intel-media.sh  DESTINATION /etc/profile.d/)
-    install (FILES ${CMAKE_CURRENT_BINARY_DIR}/cmrtlib/linux/igfxcmrt64.so DESTINATION /usr/lib/x86_64-linux-gnu/)
+
+    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/media_driver/cmake/linux/intel-media.sh.in ${CMAKE_CURRENT_SOURCE_DIR}/media_driver/cmake/linux/intel-media.sh)
+
+    install (FILES ${CMAKE_CURRENT_BINARY_DIR}/media_driver/iHD_drv_video.so DESTINATION ${CMAKE_INSTALL_LIBDIR}/dri/)
+    install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/media_driver/cmake/linux/intel-media.sh  DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/profile.d/)
```

Comment:
on Ubuntu, this CMAKE_INSTALL_SYSCONFDIR is 'etc', and the final package will put intel-media.sh under /usr/etc/profile.d/ , which is not usable to set the LIBVA search path correctly. Why the prefix '/usr' is added?

Reply:
If you define `-DCMAKE_INSTALL_PREFIX=/usr` during cmake, then CMAKE_INSTALL_SYSCONFDIR will be set to `/etc` on Ubuntu.  (See https://cmake.org/cmake/help/v3.5/module/GNUInstallDirs.html#special-cases for more details)